atlp-rwanda/commanders-rn-medica
# Feature: Doctors search and filter screens

What does this PR do?

This PR includes designs of favorite doctors, top doctors and list of all doctors where you can filter and find any doctor registered in the system of the app

Description of Task to be completed?

Add photos of the doctors and their information including reveiws and the hospitals their work on and their roles
create needed components for the designs like the cards ,buttons

How should this be manually tested?

You will click on the heart icon on the Home screen to go to favorite screen and on the favorite doctor screen then click on one of the cards where you will go to a screen of removing a doctor from the favoritates .On the home screen too you will click on the see all which is next to top doctors and you will be redirected to the top doctor screen ,there you will click on the search icon where you will be directed to the search screen which will load and after sometime you will be redirected to the not found screen and on the not found you can click on deafult you will be directed to a page where doctors were found and their list and when you click on default you will be directed to the filter screen .

Any background context you want to provide?
N/A

What are the relevant pivotal tracker/Trello stories?


Screenshots of the designs in this PR

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/33087aef-2d6d-409f-9f91-eedc2cca506b">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/a02abffd-6af3-42d2-b5ef-3dfad5f5e248">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/41368f31-0638-4915-adea-250ac50201eb">



<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/c5ee3491-7a88-4e69-873b-feaefac91656">
<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/d454ef84-7649-468a-b3c1-f11a603826c2">


<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/6179dbfa-702a-4882-86d2-d055514ca5f1">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/b289ca55-53c4-40fd-b6d5-ca2de9ca6ac5">
